### ISSUE TITLE
Private functions in Typescript are not showing in Code Map area.

### ISSUE THREAD

**Issue by maylortaylor** [oleg-shilo/codemap.vscode#2]
I see the functions that i label as "public" in the CodeMap area, but whenever I label a function as "private" i do not see it. This seems by design but would be nice to have an option to "showPrivateFunctions" option.

**Comment by Cos90** [comment]
when can support c#?

**Comment by oleg-shilo** [comment]
I am planing to release it either next week or just after Christmas break (more likely). 

Right now Syntaxer (the engine of the CS-Script plugin for Sublime Text 3) is undergoing final stage of porting for  Notepad++. And the next step is VSCode. 

And the Syntaxer is the future C# mapper for CodeMap.VSCode.

**Comment by oleg-shilo** [comment]
The solution is done. Will release it as soon as the testing is done - first thing next year :)

----

![image](https://user-images.githubusercontent.com/16729806/34028595-48108422-e1b8-11e7-93b8-6bf3a457923a.png)

**Comment by oleg-shilo** [comment]
Just published v1.2.0 with C# support

**Comment by Eisenspalter** [comment]
@oleg-shilo 
mapper_tz.zip does not work. Missing utils. `require("./utils");`

> In a mean time you I can take the attached mapper_ts.js that is the result of my quick modification of the TypeScript mapper (https://github.com/oleg-shilo/codemap.vscode/blob/master/src/mapper_ts.ts). Replace your local copy with the attached one and set that includePrivateMembers flag to true.
mapper_ts.zip

**Comment by oleg-shilo** [comment]
Yeah, I only sent the mapper file. The whole extension requires quite a few more files. I shared only the mapper but not the whole extension.

![image](https://user-images.githubusercontent.com/16729806/35834045-f291500a-0b27-11e8-8fff-e07db8c7c69c.png)

Though it's kind of irrelevant. The release is out so no need to use that zip file
